spring-kafka/src/main/java/org/springframework/kafka/annotation/EnableKafka.java

```diff
@@ -249,6 +250,6 @@
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
 @Documented
-@Import(KafkaBootstrapConfiguration.class)
+@Import({KafkaBootstrapConfiguration.class, KafkaBootstrapConfiguration.KStreamsRegistrar.class})
```

Comment:
It does a little bit too much, that's all - and IMO it's a bit of an issue that we create a `KafkaStream` and start it transparently without the user requesting it (even if apparently it doesn't do anything now, that might change). Is it at least possible to enquire whether the topology is empty or not? 

If we do it this way it is a little hard to differentiate which of the two use cases the user actually wanted (plain `Kafka` app or `KafkaStreams` app). The counterargument is that none of that would happen if the Kafka Streams library is not on the CP, so YMMV - in my head, I still think that they're two different use cases.